src/babel-plugin-bare-specifiers.ts

```diff
@@ -56,29 +57,31 @@ export const resolveBareSpecifiers = (
         return;
       }
 
-      if (isPathSpecifier(specifier)) {
+      let resolvedSpecifier;
+      try {
+        resolvedSpecifier = resolve.sync(specifier, {
+          basedir: pathlib.dirname(filePath),
+          extensions: ['.js', '.json'],
```

Comment:
Why add `.json` support? It won't do anything good in a browser.

Reply:
Fair enough, removed.

The only reason to keep it would be so that we are exactly following the Node resolution algorithm (https://nodejs.org/api/modules.html#modules_all_together).

The difference in practice if this case came up would be that the runtime error would be "Failed to load module script: The server responded with a non-JavaScript MIME type of "application/json". Strict MIME type checking is enforced for module scripts per HTML spec." instead of a 404.

Reply:
Discussed offline. We now resolve `.json` and `.node`, but we warn when you got something other than `.js`. Also added warning for when you had a bare module specifier, but it could not be resolved.